props/saber_fett263_buttons.h

```diff
@@ -466,6 +466,37 @@ enum SayType {
   SAY_MILLIS,
 };
 
+EFFECT(clrlst); // spoken color names for SAY_COLOR_LIST
+enum ColorNumber {
```

Comment:
Add a comment specifying that new colors should be added to the end and that the numbers should never change.

Reply:
Comment added